packages/data/addon/adapters/facts/interface.ts

```diff
@@ -43,6 +43,7 @@ export type SortDirection = typeof SORT_DIRECTIONS[number];
 export type ColumnType = 'metric' | 'dimension' | 'timeDimension';
 
 export type Column = {
+  cid: string;
```

Comment:
Is this required? I was thinking this would only be a PersistedColumn property as the adapter doesn't make use of it.

Reply:
For the most part no, but I do use it in the table serializer which uses the serialized version of request-v2. So I was expecting that it would be easier to keep `RequestV2` as the type also representing a serialized request fragment

Reply:
```suggestion
  cid?: string;
```